src/Pages/Index.cshtml

```diff
@@ -12,41 +12,168 @@
-        and this <a href="https://github.com/DuendeSoftware/demo.duendesoftware.com">demo server</a>.
+        This dashboard provides access to various IdentityServer functionalities for demo purposes only.
     </p>
+    <div class="d-flex align-content-between flex-wrap mb-3">
+        <div class="card m-2" style="width: 18rem;">
+            <div class="card-body">
+                <h5 class="card-title">
+                    <i class="glyphicon glyphicon-globe"></i>
+                    Discovery Document
+                </h5>
+                <p class="card-text">
+                    A discovery document
+                    where you can find metadata and links to all the endpoints, key material, etc.
+                </p>
+            </div>
+            <div class="card-footer p-0">
+                <a href="/.well-known/openid-configuration" class="btn btn-primary w-100 rounded-bottom">
+                    Go
+                    <i class="glyphicon glyphicon-chevron-right"></i>
+                </a>
+            </div>
+        </div>
+        <div class="card m-2" style="width: 18rem;">
+            <div class="card-body">
+                <h5 class="card-title">
+                    <i class="glyphicon glyphicon-user"></i>
+                    Claims
+                </h5>
+                <p class="card-text">See the claims for your current session.</p>
+            </div>
+            <div class="card-footer p-0">
+                <a href="/diagnostics/" class="btn btn-primary w-100 rounded-bottom">
+                    Go
+                    <i class="glyphicon glyphicon-chevron-right"></i>
+                </a>
+            </div>
+        </div>
+        <div class="card m-2" style="width: 18rem;">
+            <div class="card-body">
+                <h5 class="card-title">
+                    <i class="glyphicon glyphicon-book"></i>
+                    Grants
+                </h5>
+                <p class="card-text">Manage your stored grants.</p>
+            </div>
+            <div class="card-footer p-0">
+                <a href="/diagnostics/" class="btn btn-primary w-100 rounded-bottom">
+                    Go
+                    <i class="glyphicon glyphicon-chevron-right"></i>
+                </a>
+            </div>
+        </div>
+        <div class="card m-2" style="width: 18rem;">
+            <div class="card-body">
+                <h5 class="card-title">
+                    <i class="glyphicon glyphicon-paperclip"></i>
+                    Server Side Sessions
+                </h5>
+                <p class="card-text">View your current server side sessions.</p>
+            </div>
+            <div class="card-footer p-0">
+                <a href="/serversidesessions/" class="btn btn-primary w-100 rounded-bottom">
+                    Go
+                    <i class="glyphicon glyphicon-chevron-right"></i>
+                </a>
+            </div>
```

Comment:
```suggestion
```

Server side sessions are disabled in the demo - I was actually just about to remove the old link.